### ISSUE TITLE
Fallen Icon crashes game on first mob spawn

### ISSUE THREAD

**Issue by kimsis** [mariumbacchus/Soulslike-Weaponry#99]
Hello, I am using the Fantasy MC Fabric modpack, which includes Souslike-Weaponry.
I've started battling the bosses, but the Fallen Icon keeps crashing my game.
I thought it might be the java version, but it is 17.0.8, and I couldn't figure out how to change it in CurseForge.
There was one occasion where the game didn't crash on the first spawn, but it did on the 2nd spawn.
In CurseForge I have it set to use 8G of RAM, and I've tested my RAM for being faulty, and it seems to be good.

In the end I made a server, and started it and joined via localhost, I was then able to fight the boss with no issues (but for some reason my inventory isn't synced and that's a separate issue). The server starts with JRE 17.0.10+11-LTS-240, so potentially that might be the problem?

Attached below is the crash log.
[hs_err_pid30124.log](https://github.com/mariumbacchus/Soulslike-Weaponry/files/14669084/hs_err_pid30124.log)

**Comment by kimsis** [comment]
Also, once I join back my save, after a crash, my game is rolled back a few seconds before the crash (I still have my essence of eventide and am in my original spot) and the Old Moon Altar is gone.

**Comment by kimsis** [comment]
Also, it doesn't happen with the Returning Knight, Just the Fallen Icon.

**Comment by JamingDE** [comment]
i have the same problem, for me it helped to deinstall sodium, but only once lol
but i play in singleplayer

**Comment by ponduzn** [comment]
Same here, first time i summoned him it crashed just before he did a big attack. Then second time it crashed just as i spawned him.

**Comment by mariumbacchus** [comment]
I don't know why, but I can't replicate the issue, so it's hard for me to know if I've managed to fix it or not, but at least I tried. Will make an update shortly, make a new issue post then if the issue persists.
